src/documentation/0026-npm-packages-local-global/index.md

```diff
@@ -10,7 +10,7 @@ The main difference between local and global packages is this:
 - **local packages** are installed in the directory where you run `npm install <package-name>`, and they are put in the `node_modules` folder under this directory
 - **global packages** are all put in a single place in your system (exactly where depends on your setup), regardless of where you run `npm install -g <package-name>`
 
-In your code, they are both required in the same way:
+In your code, local packages are required in the same way:
```

Comment:
So an interesting issue here. Globally installed node_modules are not accessible by require by default. More details here

https://github.com/nodejs/node/blob/master/doc/api/modules.md#loading-from-the-global-folders

This text was ever accurate and we should likely fix that

Reply:
```suggestion
In your code you can only require local packages:
```